src/main/java/vectorwing/farmersdelight/blocks/BasketBlock.java

```diff
@@ -26,26 +28,58 @@
 import net.minecraft.world.World;
 
 import javax.annotation.Nullable;
-import java.util.HashMap;
-import java.util.Map;
 
 public class BasketBlock extends ContainerBlock
 {
 	public static final DirectionProperty FACING = BlockStateProperties.FACING;
 	public static final BooleanProperty ENABLED = BlockStateProperties.ENABLED;
 	public static final VoxelShape OUT_SHAPE = VoxelShapes.fullCube();
-	public static final Map<Direction, VoxelShape> SHAPE_FACING = mapVoxelShapes();
-
-	private static Map<Direction, VoxelShape> mapVoxelShapes() {
-		Map<Direction, VoxelShape> result = new HashMap<>();
-		result.put(Direction.DOWN, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(2.0D, 0.0D, 2.0D, 14.0D, 14.0D, 14.0D), IBooleanFunction.ONLY_FIRST));
-		result.put(Direction.UP, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(2.0D, 2.0D, 2.0D, 14.0D, 16.0D, 14.0D), IBooleanFunction.ONLY_FIRST));
-		result.put(Direction.NORTH, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(2.0D, 2.0D, 0.0D, 14.0D, 14.0D, 14.0D), IBooleanFunction.ONLY_FIRST));
-		result.put(Direction.SOUTH, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(2.0D, 2.0D, 2.0D, 14.0D, 14.0D, 16.0D), IBooleanFunction.ONLY_FIRST));
-		result.put(Direction.WEST, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(0.0D, 2.0D, 2.0D, 14.0D, 14.0D, 14.0D), IBooleanFunction.ONLY_FIRST));
-		result.put(Direction.EAST, VoxelShapes.combineAndSimplify(OUT_SHAPE, Block.makeCuboidShape(2.0D, 2.0D, 2.0D, 16.0D, 14.0D, 14.0D), IBooleanFunction.ONLY_FIRST));
-		return result;
-	}
+	@SuppressWarnings("UnstableApiUsage")
```

Comment:
The unstable API here is `Maps.immutableEnumMap`, but usage here can be ignored, as we are within the lifecycle of the game and the Guava API will only ever change if Mojang bump the dependency version in an update, so the usage is only as unstable as depending on game code itself.